### PR TITLE
Implements new Gandi API methods

### PR DESCRIPTION
**Pull request by Jerome1337**

### CODE REVIEW

**Review by soullivaneuh — CHANGES_REQUESTED** [review]
Coding style needs to be fixed.

**Comment by Jerome1337** [comment]
@Soullivaneuh Coding style fixed

File: src/Api/Domain.php [hunk 1]
```diff
@@ -49,6 +49,17 @@ public function getList(array $options = null): array
         return $this->getGandi()->getClient()->domain->list($options);
     }
 
+    /**
+     * @param string     $domain
+     * @param array|null $options
+     *
+     * @return array
+     */
+    final public function create(string $domain, array $options = null): array
```
Comment (soullivaneuh) [inline src/Api/Domain.php]: `?array`

File: src/Api/Domain.php [hunk 2]
```diff
@@ -59,4 +70,47 @@ public function renew(array $domain, array $options = null): array
...
+        return $this->getGandi()->getClient()->domain->delete->proceed($domain);
+    }
+
+    /**
+     * @param string $domain
+     * @param array  $options
+     *
+     * @return array
+     */
+    final public function transferIn(string $domain, array $options = null): array
```
Comment (soullivaneuh) [inline src/Api/Domain.php]: `?array`

File: src/Api/Operation.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ * @author Jérôme Pogeant <p-jerome@hotmail.fr>
+ */
+final class Operation extends AbstractApi
+{
+    /**
+     * @param int $operationId
+     *
+     * @return array
+     */
+    final public function info(int $operationId): array
```
Comment (soullivaneuh) [inline src/Api/Operation.php]: `$id` would be enough. We have enough information with the class name.

**Review by soullivaneuh — CHANGES_REQUESTED** [review]

**Comment by Jerome1337** [comment]
@Soullivaneuh requested changes fixed

**Review by soullivaneuh — APPROVED** [review]

**Comment by soullivaneuh** [comment]
I suppose it's ok. So bad we don't have tests.

**Pull request merged**